usecase/usecase.go

```diff
@@ -22,14 +22,21 @@ type dependencyManager struct {
 	gRPCClient        grpc.Client
 	responseFormatter *format.ResponseFormatter
 	resourcePresenter present.Presenter
-
-	state state
+	state             state
 }
 
+type rpcIdentifier string
+
 // state has the domain state modified by each usecase logic. The default value is used as the initial value.
 type state struct {
 	selectedPackage string // TODO: remove in v1.0.0.
 	selectedService string
+	rpcCallState    map[rpcIdentifier]callState
+}
+
+type callState struct {
+	reqPayload []byte
+	repeatable bool
```

Comment:
We can check whether an RPC is ClientStreaming or BidiStreaming by using `*grpc.RPC` (it's available in 
 `CallRPC` method in `usecase/call_rpc.go`. So, I think we don't need to store ClientStreaming/BidiStreaming states.

Reply:
Good point. It's redundant to store the "repeatable" state. However, I'm still keen on keeping the `callState` struct (even though it has a single field right now) to make it easier to add them later.

Reply:
> to make it easier to add them later.

I see. It certainly makes sense, so let's keep it that way!

Reply:
I agree with your opinion. It certainly makes sense!